pro-planner-client/src/components/AddUserForm.jsx

```diff
@@ -2,25 +2,66 @@ import React from 'react';
 import {useState} from "react";
 import {Button, Form,} from "react-bootstrap";
 import {addUser} from "../redux/userSlice";
-import {useDispatch} from 'react-redux';
+import { useDispatch, useSelector } from 'react-redux';
+
+import { setError, resetError } from '../redux/errorSlice';
+import { ERR_TYPE } from '../constants';
+
+const MAX_NAME_LIMIT = 45;
+const NAME_CHECK_REGEX = /[^0-9a-zA-Z\s]+/g;
 
 const AddUserForm = () => {
-    const [userName, setUserName] = useState('')
+    const [userName, setUserName] = useState('');
+    const users = useSelector(state => state.user.userList);
     const dispatch = useDispatch();
 
     const handleUserNameChange = (e) => {
-        setUserName(e.target.value)
+        const inputVal = e.target.value;
+        if (inputVal.length >= MAX_NAME_LIMIT) {
+            setUserName(inputVal.slice(0, MAX_NAME_LIMIT));
+        } else {
+            setUserName(e.target.value);
+        }
     }
 
     const handleNewUserSubmit = (e) => {
-        e.preventDefault()
-        if (userName !== '' && userName.trim().length > 0) {
-
-            dispatch(addUser(userName))
-
-            setUserName('')
+        e.preventDefault();
+        if (!userName) {
+            return;
         }
```

Comment:
I noticed that when I submit an empty username, the toast does not show up. Deleting these lines resolved the issue on my end.